### PR TITLE
Runner for USRP HITL test.

### PR DESCRIPTION
**Pull request by mookerji**
Command line script for running test USRP HITL test scenarios.

/cc @henryhallam @imh @mfine @denniszollo @cbeighley @fnoble @gsmcmullin

### CODE REVIEW

**Comment by henryhallam** [comment]
I think when you start the playback via ssh you can just leave that synchronous and blocking until it completes.

File: piksi_tools/testing/usrp_test.sh [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    run_parallel_pairs record_pair
+    start_scenario
+    echo "Done!"
+}
+
+function show_help () {
+    echo "Move along, nothing to see here..."
+}
+
+while getopts ":x:" opt; do
```
Comment (gsmcmullin) [inline piksi_tools/testing/usrp_test.sh]: Why bother with all this arg stuff when you can only get it wrong?  The script can only do one thing, why not just do it?

File: piksi_tools/testing/Makefile [hunk 2]
```diff
@@ -7,3 +6,5 @@ hub_ctrl: hub_ctrl.o
 clean:
 	rm -f *.o hub_ctrl
 
+usrp_test:
```
Comment (gsmcmullin) [inline piksi_tools/testing/Makefile]: I don't think this adds any value.  It doesn't build anything, and it's more obvious to just run the script than grep the makefile for the target.

File: requirements.txt [hunk 3]
```diff
@@ -9,4 +9,4 @@ pytest-cov==1.8.1
 tox==1.8.1
 virtualenv==1.11.6
 intelhex
-
+pyudev
```
Comment (gsmcmullin) [inline requirements.txt]: This appears to be unrelated.

File: piksi_tools/testing/usrp_test.sh (hunk 1 above)
Comment (mookerji) [inline piksi_tools/testing/usrp_test.sh]: If I broke into two parts (1. program., 2., run serial capture, etc.) would it make more sense then?

File: requirements.txt (hunk 3 above)
Comment (mookerji) [inline requirements.txt]: It's for cycling the hub, but it should probably be moved out of requirements and into the script.

File: piksi_tools/testing/Makefile (hunk 2 above)
Comment (mookerji) [inline piksi_tools/testing/Makefile]: Removed.

File: requirements.txt (hunk 3 above)
Comment (mookerji) [inline requirements.txt]: Removed.
